tests/unit/modules/network/vyos/fixtures/vyos_bgp_address_family_config.cfg

```diff
@@ -3,5 +3,6 @@ set protocols bgp 65536 address-family ipv4-unicast network 192.1.13.0/24 route-
 set protocols bgp 65536 address-family ipv4-unicast network 192.2.13.0/24 backdoor
 set protocols bgp 65536 address-family ipv6-unicast redistribute ripng metric '20'
 set protocols bgp 65536 neighbor 192.0.2.25 address-family ipv4-unicast route-map export 'map01'
+set protocols bgp 65536 neighbor 192.0.2.25 address-family ipv4-unicast route-map export 'map01'
```

Comment:
the same route-map for the same afi as in the previous line

Reply:
removed and tested